### ISSUE TITLE
[IntelliJ][ReportedByUser] Uncaught Exception [ba1dc57e, L:/10.102.240.103:59084 - R:springboot2.azurewebsites.net/20.118.56.7:443] Error was received while reading the incoming data. The connection will be closed. java.lang.IllegalStateException: block()/blockFirst()/blockLast() are blocking, which is not supported in thread reactor-http-nio-3

### ISSUE THREAD

**Issue by SURAJKALLOLI** [microsoft/azure-tools-for-java#6869]
IntelliJ build version: 2022.1.3 IU-221.5921.22
OS: Windows 10
JDK: JetBrains s.r.o. 11.0.15
Plugin version: 3.67.0-2022.1
Additional Info: None
Parent component: 
```
com.intellij.openapi.wm.impl.IdeRootPane[,8,8,1367x728,layout=com.intellij.openapi.wm.impl.IdeRootPane$MyRootLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=449,maximumSize=,minimumSize=,preferredSize=]
```
Error message: 
```
IdeaLoggingEvent[message=[ba1dc57e, L:/10.102.240.103:59084 - R:springboot2.azurewebsites.net/20.118.56.7:443] Error was received while reading the incoming data. The connection will be closed., throwable=java.lang.IllegalStateException: block()/blockFirst()/blockLast() are blocking, which is not supported in thread reactor-http-nio-3
	at reactor.core.publisher.BlockingSingleSubscriber.blockingGet(BlockingSingleSubscriber.java:83)
	at com.microsoft.azure.toolkit.lib.common.messager.AzureMessage.getCause(AzureMessage.java:118)
	at com.microsoft.azure.toolkit.lib.common.messager.AzureHtmlM

**Comment by wangmingliang-ms** [comment]
this bug should have been fixed by commit [bb7418](https://github.com/microsoft/azure-maven-plugins/commit/bb74188ce66b4da1294a45bcdbc92986fe3d7854) , can you try with the [latest SNAPSHOT in marketplace](https://plugins.jetbrains.com/plugin/8053-azure-toolkit-for-intellij/versions/dev).

**Comment by wangmingliang-ms** [comment]
close as it has been fixed since v3.69.0, you can upgrade to the latest and try. Feel free to reopen it when you encounter this error again.
